hypothesis-python/docs/packaging.rst

```diff
@@ -57,8 +58,9 @@ Hypothesis has *optional* dependencies on the following libraries:
 * :pypi:`Faker`, version 0.7 or later
 * `Django <https://www.djangoproject.com>`_, all supported versions
 * :pypi:`numpy`, 1.10 or later (earlier versions will probably work fine)
-* :pypi:`pandas`, 1.8 or later
-* :pypi:`py.test <pytest>` (2.8.0 or greater). This is a mandatory dependency for testing Hypothesis itself but optional for users.
+* :pypi:`pandas`, 1.19 or later
+* :pypi:`pytest` (2.8.0 or greater). This is a mandatory dependency for
```

Comment:
Is it 2.8 or 3.0?

Reply:
Whoops, I wrote that before discovering that 2.8 doesn't work :disappointed: